### ISSUE TITLE
Potential support for CRAM input

### ISSUE THREAD

**Issue by HudoGriz** [KCCG/ClinSV#26]
Hello ClinSV team!

Great software, we include it regular in our new workflows!
We would like to re-analyse old data, but the alignments are compressed in CRAM files. We are sure that there are more labs which keep the data as CRAM (less space than BAM), so we suggest including CRAM files as optional input files.

Keep up the great work!

Blaž Vrhovšek

**Comment by melnel000** [comment]
Thanks ClinSV for sharing this pipeline. I am about to try it out but I also need to uncompress my CRAM files to BAM so CRAM support would be a great feature.

**Comment by drmjc** [comment]
Hi, thanks for the great feedback. 
Adding seamless support for different ref genomes is at the top of the list, but I agree this will be a useful feature to add

**Comment by HudoGriz** [comment]
@melnel000 out of curiosity, did you manage to run the analysis on converted files?
For me CNVnator fails on converted BAM files. The conversion was done using _Samtools_: `samtools view -b  -T <refgenome.fa> -o <file.bam> <file.cram>`.

**Comment by melnel000** [comment]
@HudoGriz yes, I could run on the bam files without issues and some of these were converted using samtools.

**Comment by cluhaowie** [comment]
Hi, all I just have the same suggestion with the cram file and see people have already posted it earlier. Wonder if it's any update?

**Comment by drmjc** [comment]
Hi, thanks for the feature request. It's a very  sensible one. We've just
switched to cram too so this has got a chance of getting done, but we'll
need to assess how much work it'll take to implement this, so I don't have
a timeline for you.
Cheers
